### PR TITLE
add a swipe refresh layout on notifications page #248

### PR DESCRIPTION
**Pull request by syedjahangirpeeran**

### CODE REVIEW

File: Android/app/src/main/java/io/github/project_travel_mate/notifications/NotificationsActivity.java [hunk 1]
```diff
@@ -70,7 +73,12 @@ protected void onCreate(Bundle savedInstanceState) {
         notifications = new ArrayList<>();
 
         getNotifications();
-
+        mySwipeRefreshLayout.setOnRefreshListener( new SwipeRefreshLayout.OnRefreshListener() {
+            @Override
+            public void onRefresh() {
+                getNotifications();
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/notifications/NotificationsActivity.java]: show the animation by the time API is being called.

Comment (syedjahangirpeeran) [reply Android/app/src/main/java/io/github/project_travel_mate/notifications/NotificationsActivity.java]: Shall I remove the default circular progress bar which we get ?

Comment (Swati4star) [reply Android/app/src/main/java/io/github/project_travel_mate/notifications/NotificationsActivity.java]: Keep the animation.

If you feel like the circular progress bar doesn't look good above the animation, you can remove that.

**Comment by syedjahangirpeeran** [comment]
Done!

**Pull request merged**